### DISCUSSION TITLE
登录节点失败的问题

### DISCUSSION THREAD

**Discussion by Nickel-x** [natfrp/wiki#129] [使用问题]
ping域名ping不通，ping ip却ping得通，这应该是dns的问题？但我换了几个dns都不行，阿里云公共dns和其他一些试过不行，请问该怎么解决呢
![image](https://user-images.githubusercontent.com/110629765/182926950-256ad501-67c5-4927-a71b-adbc9b5fca0d.png)

...

**Comment by Nickel-x** [comment]
![image](https://user-images.githubusercontent.com/110629765/182928609-a2b8e0be-4c6f-48f8-85e7-df7b0031bbfc.png)
当我创建隧道的时候出现的错误是这样，这里用的是#77 南宁电信1

**Reply by Liner03** [reply]
请问解决没
